egs/iban/s5/local/online/run_nnet2_multisplice.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
```

Comment:
I think it might be better to move to nnet3. We are no longer "supporting" this version.